.artifact/tree.json

```diff
@@ -51303,99 +51303,99 @@
         },
         "position": {
           "position_0.json": {
-            "hash": "e69477a7744bccdecb2b080c552f993b",
-            "size": 255
+            "hash": "c1d032d78232e432d577566e5d5a33b1",
+            "size": 260
           },
           "position_1.json": {
-            "hash": "25df91a467e83eec4a210e1b3c4ebdcd",
+            "hash": "58f3dea23ca043b262eb8dd21a419247",
```

Comment:
I'm not sure why these JSONs changed, but it might be better to keep the hashes what they were and only add the one new entry for the tflite file into tree.json